ci/Jenkinsfile.tests-e2e

```
@@ -97,10 +97,10 @@ pipeline {
 
     stage('Client') {
       environment {
-        GANACHE_NETWORK_RPC_URL = "http://localhost:${env.GANACHE_RPC_PORT}"
+        STATUS_RUNTIME_GANACHE_NETWORK_RPC_URL = "http://localhost:${env.GANACHE_RPC_PORT}"
       }
       steps { script {
-        sh 'make nim_status_client'
+        linux.bundle('tgz-linux')
```

This is wrong, it should be:
```suggestion
        linux.bundle('nim_status_client')   
```
Made the change separately to verify it works:

* https://github.com/status-im/status-desktop/pull/12531